ui/pages/destinations/add.js

```diff
@@ -238,11 +255,12 @@ export default function DestinationsAdd() {
               </h3>
             </div>
           )}
-          <Link href='/destinations'>
-            <a className='flex-none items-center self-center rounded-md border border-transparent bg-black px-4 py-2 text-2xs font-medium text-white shadow-sm hover:bg-gray-800'>
-              Finish
-            </a>
-          </Link>
+          <button
+            className='inline-flex items-center rounded-md border border-transparent bg-black px-4 py-2 text-2xs font-medium text-white shadow-sm hover:cursor-pointer hover:bg-gray-800 disabled:cursor-not-allowed disabled:opacity-30'
+            onClick={finish}
```

Comment:
Oh! this is such a good idea!!
nit: we don't really need a finish function. we can put all that in `onClick={async() => { ... }}`

Reply:
and we need a type in button tag

Reply:
thanks for the feedback, made these changes